src/py42/clients/alertrules/exfiltration.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from py42.clients import BaseClient
+
+
+class ExfiltrationClient(BaseClient):
+
+    _version = u"v1"
+    _resource = u"query-endpoint-exfiltration-rule"
+    _api_prefix = u"/svc/api/{0}/Rules/{1}".format(_version, _resource)
+
+    def __init__(self, session, tenant_id):
+        super(ExfiltrationClient, self).__init__(session)
+        self._tenant_id = tenant_id
+
+    def get(self, rule_id):
+        """Fetch exfiltration alert rule by rule id.
+
+        Args:
+            rule_id (str): Observer rule Id of a rule to be fetched.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        data = {"tenantId": self._tenant_id, "ruleIds": [rule_id]}
```

Comment:
Missing `u""s here

Reply:
The three muska-rule-type clients all have this here

Reply:
I am sorry, this u"" thing is coming up again and again and I feel like I have fixed all, I wonder if there was command which could let us know if all string are python 2 compatible, I would add it to `pre-commit` . Fixed.